cache/src/main/java/com/uber/buckcache/datastore/impl/ignite/LocalCacheEventListener.java

```diff
@@ -27,6 +28,14 @@ public LocalCacheEventListener(IgniteCache<Long, byte[]> metadataCache, IgniteCa
 
   @Override
   public boolean apply(CacheEvent evt) {
+    if (evt.cacheName().equals(METADATA_CACHE_NAME)) {
```

Comment:
What about the other caches?

Reply:
`EXPIRED`, `REMOVED` and `DESTROYED` will get triggered for all 3 caches at the same time I believe. There is no point tracking all 3 of them.
`EVICTED` is something we may need to track separately for each cache, we can do that later.

I can prefix the metric-name with cache-name so that they can be tracked separately in future.

What are your thoughts?

Reply:
@anoopc1 Yeah, if it's not too much extra data, maybe we can track each separately and get some insight from there.

Reply:
Updated. PTAL @mmdango